### PR TITLE
Add Ellaism coin in chain config

### PR DESCRIPTION
**Pull request by ellaismer**
rel https://github.com/ellaism/parity-config/issues/5

Added chain config so Ellaism users can use Parity without downloading another config file.

### CODE REVIEW

**Comment by ellaismer** [comment]
[clabot:check]

File: ethcore/res/ethereum/ellaism.json [hunk 1]
```diff
@@ -4,15 +4,13 @@
   "engine": {
     "Ethash": {
       "params": {
-        "gasLimitBoundDivisor": "0x0400",
```
Comment (5chdn) [inline ethcore/res/ethereum/ellaism.json]: This should not be removed but go down to the general `params` section below.

File: ethcore/res/ethereum/ellaism.json [hunk 2]
```diff
@@ -4,15 +4,13 @@
...
         "minimumDifficulty": "0x020000",
         "difficultyBoundDivisor": "0x0800",
         "durationLimit": "0x0d",
         "blockReward": "0x4563918244F40000",
         "homesteadTransition": "0x0",
         "bombDefuseTransition": "0x0",
         "eip150Transition": "0x0",
-        "eip155Transition": "0x0",
```
Comment (5chdn) [inline ethcore/res/ethereum/ellaism.json]: Should also go down to the general `params` section.

**Comment by ellaismer** [comment]
@5chdn "gasLimitBoundDivisor" and "eip155Transition" is in the general "params" section, in L23 and L29.

**Pull request merged**